client/src/components/DropdownSelect.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import Typography from '@material-ui/core/Typography'; 
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+    menuPaper: {
+        maxHeight: 200,
+      },
+    
+    text: {
+        fontSize: 14,
+      }
+}))
+
+function DropdownSelect({defaultValue,handler,options}) {
+
+    const classes = useStyles();
+
+    return (       
+            <TextField
+                select 
+                SelectProps={{ 
+                    MenuProps: {classes: { paper: classes.menuPaper}}  
+                }}
```

Comment:
It's styling applied to the selection box when you click on the drop down. The menu component is within the select component. I have to do this since i want to style a deeper component for the Material UI textfield component.